index.bs

```diff
@@ -1703,8 +1732,14 @@ When this method is invoked, the [=authenticator=] must perform the following pr
 - Prompt the user to select a [=public key credential|credential=] from among the above list. Obtain [=user consent=] for using
     this [=public key credential|credential=]. The prompt for obtaining this [=user consent|consent=] may be shown by the
     [=authenticator=] if it has its own output capability, or by the user agent otherwise.
-- Process all the supported extensions requested by the client, and generate the [=authenticator data=] as specified in
-    [[#sec-authenticator-data]], though without [=attestation data=]. Concatenate this [=authenticator data=] with the [=hash of
+- Process all the supported extensions requested by the client.
+- If the [=signature counter=] feature is supported, increment the [=RP ID=]-associated 
+    [=signature counter=] or the global [=signature counter=] value, depending on 
+    which approach is implemented by the [=authenticator=] by some positive value.  Otherwise, use a value 
+    of zero (0) for the [=signature counter=] value when generating the [=authenticator data=].
```

Comment:
If we want to implement Alex's request to *require* the counter, we also need to remove the option to unconditionally return 0.